plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/point/commonStyles/index.ts

```diff
@@ -42,17 +43,23 @@ export const FieldWrapper = styled.div<{ gap?: number }>`
 
 export const FieldValue = styled.div`
   display: flex;
+  justify-content: start;
+  align-items: center;
   border: 1px solid #d9d9d9;
   border-radius: 2px;
   flex: 1;
   height: 100%;
   width: 100%;
 `;
 
+export const BorderLessFieldValue = styled(FieldValue)`
```

Comment:
BorderlessFieldValue
L doesn't need to be capitalized

Reply:
Done! added `noBorder` prop to `FieldValue`.